extension/solarwindsextension/README.md

```diff
@@ -0,0 +1,41 @@
+# Solarwinds Extension
+
+<!-- status autogenerated section -->
+| Status        |           |
+| ------------- |-----------|
+| Stability     | [development]  |
+| Distributions | [] |
+| Issues        | [![Open issues](https://img.shields.io/github/issues-search/solarwinds/solarwinds-otel-collector?query=is%3Aissue%20is%3Aopen%20label%3Aextension%2Fsolarwinds%20&label=open&color=orange&logo=opentelemetry)](https://github.com/solarwinds/solarwinds-otel-collector/issues?q=is%3Aopen+is%3Aissue+label%3Aextension%2Fsolarwinds) [![Closed issues](https://img.shields.io/github/issues-search/solarwinds/solarwinds-otel-collector?query=is%3Aissue%20is%3Aclosed%20label%3Aextension%2Fsolarwinds%20&label=closed&color=blue&logo=opentelemetry)](https://github.com/solarwinds/solarwinds-otel-collector/issues?q=is%3Aclosed+is%3Aissue+label%3Aextension%2Fsolarwinds) |
+
+[development]: https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/component-stability.md#development
+<!-- end autogenerated section -->
+
+SolarWinds Extension is required sidecar for the [Solarwinds Exporter](../../exporter/solarwindsexporter).
```

Comment:
I'd be cautious about the term "sidecar" here - it could be confusing to our users with a K8s background where it means a specific thing.